tutorials/R/Geospatial-skills/primer-raster-r/Image-Raster-Data-In-R/Image-Raster-Data-In-R.Rmd

```diff
@@ -332,7 +347,8 @@ orderRGBstack <- stack(rgbRaster$band58,rgbRaster$band34,rgbRaster$band19)
 
 # write the geotiff
 # change overwrite=TRUE to FALSE if you want to make sure you don't overwrite your files!
-writeRaster(orderRGBstack,"rgbRaster.tif","GTiff", overwrite=TRUE)
+writeRaster(orderRGBstack,paste0(wd,"NEON-DS-Field-Site-Spatial-Data/SJER/RGB/rgbRaster.tif"),"GTiff", overwrite=TRUE)
```

Comment:
Nice work! Looks like you made all of the correct paste0() substitutions throughout this .Rmd!